bin/ConcordanceMakeSamplesheet.sh

```diff
@@ -185,10 +185,10 @@ do
 	ngsFolderPrm="$(dirname "${vcfFile}")"
 	ngsVcfId=$(basename "${vcfFile}" ".final.vcf.gz")
 	# shellcheck disable=SC2029
-	if ssh "${HOSTNAME_PRM}" "zcat \"${vcfFile}\" | grep \'##FastQ_Barcode=\'"
+	if ssh "${HOSTNAME_PRM}" "zcat \"${vcfFile}\" | grep \""##FastQ_Barcode=\""
 	then
 		# shellcheck disable=SC2029
-		ngsBarcodeTmp=$(ssh "${HOSTNAME_PRM}" "zcat \"${vcfFile}\" | grep \'##FastQ_Barcode=\'")
+		ngsBarcodeTmp=$(ssh "${HOSTNAME_PRM}" "zcat \"${vcfFile}\" | grep \""##FastQ_Barcode=\"")
```

Comment:
De dubbele quotes moesten ge-escaped worden, maar de enkele niet. Dus de simpeler oplossing is:
```
ngsBarcodeTmp=$(ssh "${HOSTNAME_PRM}" "zcat \"${vcfFile}\" | grep '##FastQ_Barcode='")
```